templates/web/base/waste/bin_days.html

```diff
@@ -8,6 +8,19 @@ <h1 class="govuk-heading-xl">Your bin days</h1>
 
 [% INCLUDE 'waste/_address_display.html' %]
 <div class="waste__collections">
+
+[% IF property.has_pending_bulky_collection %]
```

Comment:
Not sure how this should look. It looks ugly :-/

Reply:
This should probably be in the `bin_days_bulky.html` template as that's what's used instead of this one.